src/main/java/springboot/Controller/ItemController.java

```diff
@@ -32,13 +32,7 @@ private ItemController(ItemService service, RubricService rubricService, PersonS
 
     @GetMapping(value = "/addItemPage")
     public String addItemPage(@ModelAttribute Item item, Model model) {
-        Boolean authorized = false;
-        for(Person p : personService.getAllPersons()) {
-            if(p.isUser() && (p.getType().equals(Person.Type.INSTRUCTOR.toString()))) {
-                authorized = true;
-            }
-        }
-        if(!authorized){ return "unauthorizedUser"; }
+        if(!personService.isAuthorized()){ return "unauthorizedUser"; }
```

Comment:
Minor thing, but in net security he talked about how the positive authorization should be the if statement. And then the if not authorized stuff can come after